src/skeletongraphicswidget.cpp

```diff
@@ -910,6 +910,36 @@ bool SkeletonGraphicsWidget::mouseMove(QMouseEvent *event)
         }
     }
     
+    if (SkeletonDocumentEditMode::Select == m_document->editMode) {
+        if (QGuiApplication::queryKeyboardModifiers().testFlag(Qt::ShiftModifier)) {
+            if(m_scaleStarted) {
+                QPoint currentPos = event->globalPos();
+                QPoint deltaPos = currentPos - m_lastScalePos;
+                m_lastScalePos = currentPos;
+                float delta = deltaPos.x() * 0.05;
+                if (!m_rangeSelectionSet.empty()) {
+                    if (QGuiApplication::queryKeyboardModifiers().testFlag(Qt::ControlModifier)) {
+                        scaleSelected(delta);
+                    } else {
+                        zoomSelected(delta);
+                    }
+                    emit groupOperationAdded();
```

Comment:
emit groupOperationAdded only when the scale is done to avoid every fine move operation insert into the undo queue.